type-definitions/tests/immutable-flow.js

```diff
@@ -5,7 +5,7 @@
 // Some tests look like they are repeated in order to avoid false positives.
 // Flow might not complain about an instance of (what it thinks is) T to be assigned to T<K, V>
 
-import Immutable, {
```

Comment:
I made it - I know it's confusing. It tests if (1) the default import (2) the `import * as X from 'immutable'` (3) individual imports are supported.